x-pack/legacy/plugins/reporting/server/config/index.ts

```diff
@@ -109,6 +109,7 @@ export interface ReportingConfigType {
     checkForFormulas: boolean;
     maxSizeBytes: number;
     useByteOrderMarkEncoding: boolean;
+    escapeFormulaValues: boolean;
```

Comment:
The changes here and in the config.ts file will conflict with https://github.com/elastic/kibana/pull/62500 and I'm mere minutes away from merging that. Sorry! But I think we should merge the older PR first.

When those conflicts are resolved, the new setting added here will be defined in the New Platform Reporting plugin :)